File: 13-sorting/1-intro/main.java
// Sorting Introduction - Java Example

import java.util.Arrays;

public class SortingIntro {
    public static void main(String[] args) {
        int[] arr = {5, 3, 8, 4, 2};
        int[] sortedArr = arr.clone();
        Arrays.sort(sortedArr);

        System.out.print("Original array: ");
        for (int x : arr) System.out.print(x + " ");
        System.out.print("\nSorted array: ");
        for (int x : sortedArr) System.out.print(x + " ");
        System.out.println();
    }
}

File: 13-sorting/1-intro/main.cpp
// Sorting Introduction - C++ Example

#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    std::vector<int> arr = {5, 3, 8, 4, 2};
    std::vector<int> sorted_arr = arr;
    std::sort(sorted_arr.begin(), sorted_arr.end());

    std::cout << "Original array: ";
    for (int x : arr) std::cout << x << " ";
    std::cout << "\nSorted array: ";
    for (int x : sorted_arr) std::cout << x << " ";
    std::cout << std::endl;
    return 0;
}
